packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -442,7 +457,6 @@ function UserMenu({
             {isGovernment ? 'Cloud for Government' : 'Cloud'}
           </MenuItem>
         )}
-
```

Comment:
Looks like my Prettier auto-removed these extra spaces, but I also didn't see any issue/changes when running `yarn prettier:fix` so let me know if I need to change this back.